aiida_sssp_workflow/workflows/evaluate/_cohesive_energy.py

```diff
@@ -170,8 +170,18 @@ def inspect_energy(self):
             output_parameters = child.outputs.output_parameters
 
             atom_free_energy = output_parameters["energy"]
-            atom_smearing_energy = output_parameters["energy_smearing"]
-            atom_energy = atom_free_energy - atom_smearing_energy
+            if output_parameters["smearing_type"] in ["gaussian", "fd"]:
+                # need to interplate to T=0K
+                atom_smearing_energy = output_parameters["energy_smearing"]
+                atom_energy = atom_free_energy - atom_smearing_energy
```

Comment:
```suggestion
                atom_energy = atom_free_energy - 0.5 * atom_smearing_energy
```
Check with @flavianojs